consumers/src/unit-test/java/uk/gov/moj/sdt/cmc/consumers/client/xml/XmlReaderTest.java

```diff
@@ -38,4 +40,13 @@ private void readXmlFileAndValidateData(String fileName, String xmlNodeName, Str
         assertEquals(expectedValue, returnValue);
     }
 
+    @Test
+    void shouldConvertClaimStatusUpdateRequestToString() throws IOException {
```

Comment:
This test should use the readXmlFileAndValidateData() method rather than duplicating the contents of it.

Reply:
done